src/main/java/com/zendesk/maxwell/schema/columndef/BitColumnDef.java

```diff
@@ -16,7 +16,12 @@ public boolean matchesMysqlType(int type) {
 
 	@Override
 	public Object asJSON(Object value) {
-		byte[] bytes = (byte[]) value;
+		byte[] bytes;
+		if( Boolean.class.isAssignableFrom(value.getClass()) ){
```

Comment:
actually i take that back, null should probably not trickle through safely here since it could produce an npe.  i'll tighten it up with the instance check